docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,162 @@
+
+   .. image:: ../images/gisaid-select-records.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+5. Select **Download** in the bottom right of the search results.
+6. Select **Input for the Augur pipeline** as the download format.
+
+   .. image:: ../images/gisaid-augur-pipeline-download.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You may see different download options, but it is fine as long as **Input for the Augur pipeline** is available.
+
+7. Select **Download**.
+8. Download/move the ``.tar`` file into the ``data/`` directory in the ``ncov/`` directory.
+9. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+10. Rename the files as ``custom.metadata.tsv`` and ``custom.sequences.fasta``.
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a pre-written ``--configfile``:
+
+.. code:: text
+
+   nextstrain build . --cores all --configfile ncov-tutorial/custom-data.yaml
+
+Break down the command
+~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow can take several minutes to run. While it is running, you can investigate the contents of ``custom-data.yaml`` (comments excluded):
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       sequences: https://data.nextstrain.org/files/ncov/open/reference/sequences.fasta.xz
+     - name: custom_data
+       metadata: data/custom.metadata.tsv
+       sequences: data/custom.sequences.fasta
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     custom-build:
+       title: "Build with custom data and example data"
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+This is the same as the previous file, with some additions:
+
+1. A second input for the custom data, referencing the metadata and sequences files downloaded from GISAID.
+2. A ``builds`` section that defines one output :term:`docs.nextstrain.org:dataset` using:
+
+   1. A custom name ``custom-build``
+   2. A custom title ``Build with custom data and example data``
+   3. An Auspice config file with the contents:
+
+      .. code-block:: json
+
+         {
+           "colorings": [
+             {
+               "key": "custom_data",
+               "title": "Custom data",
+               "type": "categorical"
+             }
+           ],
+           "display_defaults": {
+             "color_by": "custom_data"
+           }
+         }
+
+      This JSON does two things:
+
+      1. Create a new coloring ``custom_data`` which reflects a special metadata column generated by the ncov workflow. Each data input produces a new final metadata column with categorical values ``yes`` or ``no`` representing whether the sequence was from the input.
```

Comment:
Originally from @huddlej: https://github.com/nextstrain/ncov/commit/672d2055794ac803e11f957d8b5833fda2a87521#r69298892

> Each data input produces a new final metadata column with categorical values...

The workflow only adds these input-specific columns when there are multiple inputs during [the `combine_input_metadata` rule](https://github.com/nextstrain/ncov/blob/464041debc66a86a33d3c1c697026a1adeb1d326/workflow/snakemake_rules/main_workflow.smk#L35-L58). We may need to clarify this behavior here, so users aren't confused when they look for a similar column with a single input (not that such a column would be useful!).

Reply:
Done: ac84d6eb